### PR TITLE
add Set_operators demonstration

### PR DESCRIPTION
**Pull request by hz66-404**

### CODE REVIEW

**Comment by N-J-Martin** [comment]
looks good so far! Is there anything more you want to add?

**Comment by hz66-404** [comment]
Every set operator has done, if everything is fine, help me to merge to the main conjure branch! thank you so much!

File: docs/bits/operator/L_in.md
Comment (N-J-Martin) [inline docs/bits/operator/L_in.md:1]: The links should end with docs/notebooks/Set_operators.ipynb

**Review by N-J-Martin — COMMENTED** [review]
Just the links need correcting. The rest look great!

**Comment by hz66-404** [comment]
Thanks! have already changed all

**Review by N-J-Martin — APPROVED** [review]
looks good. thank you!

**Pull request merged**